### PR TITLE
[Docs] refactor(docs): assets migration (2/3)

### PR DESCRIPTION
**Pull request by carmen279**
# Pull request template

This PR brings to this repository all the assets used in the doc contained here. It is the second step on a 3 step process to fulfill this migration.
⚠️ <strong>WARNING </strong> ⚠️ For this PR to work without any issue, [this one](https://github.com/empathyco/docs-empathy/pull/1010) has to be merged first.

## Motivation and context
This PR brings to this repository all the assets used in the docs.

- [X] Dependencies. If any, specify: [PR from eDocs](https://github.com/empathyco/docs-empathy/pull/1010)
- [ ] Open issue. If applicable, link:

## Type of change

- [ ] Bug fix (non-breaking change that fixes an issue)
- [ ] New feature (non-breaking change that adds functionality)
- [ ] Breaking change (fix or feature that causes existing functionality to not work as expected)
- [ ] Change requires a documentation update

## What is the destination branch of this PR?

- [X] `Main`
- [ ] Other. Specify:

## How has this been tested?

Tests performed according to [testing guidelines](./contributing/tests.md): 

## Checklist:

- [X] My code follows the **[style guidelines](./CONTRIBUTING.md#style-guides)** of this project.
- [X] I have performed a **self-review** on my own code.
- [X] My changes generate **no new warnings**.

### CODE REVIEW

File: packages/x-components/static-docs/build-search-ui/README.md [hunk 1]
```diff
@@ -20,7 +20,7 @@ Start building your search and discovery experience:
 
 ::: slot right-intro-column
 
-<img :src="$withBase('/assets/media/interface/build-search-ui.svg')" alt="Build Your Search UI">
+<img :src="$withBase('/assets/x/interface/build-search-ui.svg')" alt="Build Your Search UI">
```
Comment (javieri-empathy) [inline packages/x-components/static-docs/build-search-ui/README.md:23]: Just curious, why some images are referenced through an `img` tag with a `$withBase` helper, but others are  used with a markdown image and a `~` path?

File: packages/x-components/static-docs/experience-search-and-discovery/history-queries.md [hunk 2]
```diff
@@ -5,7 +5,7 @@ title: History Queries UI
 The History Queries UI component displays the search history, allowing shoppers to check previous
 searches and revisit products they’ve viewed before in a few clicks.
 
-![History Queries](~@assets/media/interface/x-history-queries.gif)
+![History Queries](~@assets/media/x/x-history-queries.gif)
```
Comment (javieri-empathy) [inline packages/x-components/static-docs/experience-search-and-discovery/history-queries.md:8]: Is this path correct? It's different from the previous ones

```suggestion
![History Queries](~@assets/x/interface/x-history-queries.gif)
```

**Review by javieri-empathy — CHANGES_REQUESTED** [review]

File: package-lock.json [hunk 3]
```diff
@@ -11885,9 +11885,9 @@
       }
     },
     "node_modules/immutable": {
-      "version": "4.2.1",
-      "resolved": "https://registry.npmjs.org/immutable/-/immutable-4.2.1.tgz",
-      "integrity": "sha512-7WYV7Q5BTs0nlQm7tl92rDYYoyELLKHoDMBKhrxEoiV4mrfVdRz8hzPiYOzH7yWjzoVEamxRuAqhxL2PLRwZYQ=="
+      "version": "4.2.2",
```
[the remark this answers is not in the document]
Comment (carmen279) [reply package-lock.json]: Sorry, my mistake, it was an accident.

File: packages/x-components/static-docs/experience-search-and-discovery/history-queries.md (hunk 2 above)
Comment (carmen279) [reply packages/x-components/static-docs/experience-search-and-discovery/history-queries.md]: You are right!

**Comment by support-empathy** [comment]
This is a link to Empathy Platform Docs Portal preview in a development environment. 
 ⚠️ Only for Empathy members. We’re sorry for the inconvenience!
 [Check out EPDocs preview](https://docs-dev.empathy.co/preview/x-989/index.html)

**Pull request closed without merging**